src/go/k8s/tests/e2e/pandaproxy-produce-consume-tls-client-external-ca/00-assert.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: cluster-proxy
+status:
+  replicas: 1
```

Comment:
I would advise to use something like https://github.com/redpanda-data/redpanda/blob/a628e95cf23f1d60a29b2d7313f688ca975cac74/src/go/k8s/tests/e2e/update-image-tls/00-assert.yaml#L74-L86 to assert Cluster is created and configured.